<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_stmanager_detail</class>
 <widget class="QWidget" name="admin_stmanager_detail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AtQlas Alpha</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>290</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>211</red>
         <green>224</green>
         <blue>229</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>211</red>
         <green>224</green>
         <blue>229</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>127</red>
         <green>127</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>PT Mono</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border-image:url(:/atqres_ico/pure_gray_7.png)
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>290</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>211</red>
         <green>224</green>
         <blue>229</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>211</red>
         <green>224</green>
         <blue>229</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>127</red>
         <green>127</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>PT Mono</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border-image:url(:/atqres_ico/pure_gray_7.png)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="school_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>290</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>211</red>
         <green>224</green>
         <blue>229</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>211</red>
         <green>224</green>
         <blue>229</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>127</red>
         <green>127</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>PT Mono</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border-image:url(:/atqres_ico/pure_gray_7.png)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="klass_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>290</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>211</red>
         <green>224</green>
         <blue>229</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>211</red>
         <green>224</green>
         <blue>229</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>127</red>
         <green>127</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>PT Mono</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border-image:url(:/atqres_ico/pure_gray_7.png)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="grade_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>290</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>211</red>
         <green>224</green>
         <blue>229</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>211</red>
         <green>224</green>
         <blue>229</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>127</red>
         <green>127</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>PT Mono</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border-image:url(:/atqres_ico/pure_gray_7.png)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lesson_chooser_edit">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>130</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PT Mono</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border-image:url(:/atqres_ico/pure_gray.png)</string>
   </property>
   <property name="placeholderText">
    <string>Filter...</string>
   </property>
  </widget>
  <widget class="QListWidget" name="recommendation">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>0</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border-image:url(:/atqres_ico/pure_gray.png)</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>506</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border-image:url(:/atqres_ico/admin_stmanager.png)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTableWidget" name="opt_lesson">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>651</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget 
{background-color: transparent;}
QScrollBar:vertical {
    background: url(images/pure_blue.png);
    width: 0px;
    margin: 0px 0 0px 0;
}
QScrollBar::handle:vertical {
    background: rgb(195, 195, 195);
    min-height: 30px;
    margin: 0 1px 0 2px;
    border-radius: 3px;
    border: none;
}

QScrollBar:horizontal {
    height: 0px;
    margin: 0px 0 0px 0;
}
QScrollBar::handle:horizontal {
    background: rgb(195, 195, 195);
    min-width: 20px;
    margin: 2px 0px 1px 0px;
    border-radius: 3px;
    border: none;
}
QHeaderView::section
{
border-image:url(:/atqres_ico/pure_gray.png);
height:30px;
spacing: 20px;
background-color:rgb(39,49,58);
color: white;

text-align: right;
font-family: arial;
font-size:12px;
}</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="req_lesson">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>651</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget 
{background-color: transparent;}
QScrollBar:vertical {
    background: url(images/pure_blue.png);
    width: 0px;
    margin: 0px 0 0px 0;
}
QScrollBar::handle:vertical {
    background: rgb(195, 195, 195);
    min-height: 30px;
    margin: 0 1px 0 2px;
    border-radius: 3px;
    border: none;
}

QScrollBar:horizontal {
    height: 0px;
    margin: 0px 0 0px 0;
}
QScrollBar::handle:horizontal {
    background: rgb(195, 195, 195);
    min-width: 20px;
    margin: 2px 0px 1px 0px;
    border-radius: 3px;
    border: none;
}

QHeaderView::section
{
border-image:url(:/atqres_ico/pure_gray.png);
height:30px;
spacing: 20px;
background-color:rgb(39,49,58);
color: white;

text-align: right;
font-family: arial;
font-size:12px;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_lesson_button">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>270</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border-image:url(:/atqres_ico/gray_minus.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="add_lesson_button">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>270</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border-image:url(:/atqres_ico/login_plus.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <zorder>frame</zorder>
  <zorder>lineEdit_4</zorder>
  <zorder>name_edit</zorder>
  <zorder>school_edit</zorder>
  <zorder>klass_edit</zorder>
  <zorder>grade_edit</zorder>
  <zorder>lesson_chooser_edit</zorder>
  <zorder>recommendation</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
